requirements/base.txt

```diff
@@ -0,0 +1,14 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file base.txt base.in
+#
+aiohttp==1.0.5            # via discord.py
+async-timeout==2.0.0      # via aiohttp
+chardet==3.0.4            # via aiohttp
+discord.py==0.16.12
+multidict==3.3.2          # via aiohttp
+pyyaml==3.12
+urwid==1.3.1
+websockets==3.4           # via discord.py
```

Comment:
should this file be here or autogenerated locally? just wondering

Reply:
It must be here since the versions are frozen to avoid regressions over time (I had such an issue with a old project when I did not specify the package version and when they relased another version with breaking changes)

Reply:
ah cool then!